### PR TITLE
q40603 hw3 attempt1

### PR DESCRIPTION
**Pull request by q40603**
---
A good PR should include the following parts:

- [ ] A clear title
- [ ] A meaningful message for PR, as well as its commits
- [ ] from your specific branch (***not master***) merging to teacher's `HW` branch (***not master***, again)
- [ ] excluding any irrelevant files, such as binaries, text files, or dot files
- [ ] passing tests/CI

### CODE REVIEW

**Comment by q40603** [comment]
CI fail. 哭哭

**Comment by tigercosmos** [comment]
you should send PR to `hw3`

File: hw3/q40603/validate.py [hunk 1]
```diff
@@ -0,0 +1,249 @@
+#!/bin/bash
```
Comment (tigercosmos) [inline hw3/q40603/validate.py:1]: you don't need to copy this.

File: hw3/q40603/.gdb_history [hunk 2]
```diff
@@ -0,0 +1,9 @@
+run
```
Comment (tigercosmos) [inline hw3/q40603/.gdb_history]: you don't need to upload garbage.

**Pull request closed without merging**